src/DateClass/Kiemtrangaythang.java

```diff
@@ -0,0 +1,45 @@
+package DateClass;
+
+public class Kiemtrangaythang {
+	private int year;
+	private int month;
+	private int day;
+
+	public Kiemtrangaythang(int year, int month, int day) {
+		if (year <= 0 || month <= 0 || (day <= 0 && day > 31)) {
+			System.out.println("Ngay thang nam nhap vao khong hop le");
+			return;
+		}
+		System.out.println("Ngay thang nam nhap vao hop le");
+	}
+
+	public void kiemtraNamNhuan(int year, int month, int day) {
+		if (month == 2 && year % 4 == 0 || year % 400 == 0) {
+			if (day == 29) {
+				System.out.println("Nam" +year+ "la nam nhuan");
+				return;
+			}
+			if (day > 29) {
+				System.out.println("Ngay thang nam nhap vao hop le");
+				return;
+			}
+			System.out.println("Nam" +year+ "khong phai la nam nhuan");
```

Comment:
Có vài chỗ ko hợp lý lắm nè:
1. ủa if - line 17 thì đã là năm nhuận mà line 26 lại in ra đây ko phải năm nhuận thì hok hợp lý nha
2. nếu chị nhập ngày 3 tháng 2 của năm nhuận thì nó là năm nhuận mà cu :v
```suggestion
   if(day > 29) {
         System.out.println("Ngay thang nam nhap vao KHONG hop le");
	 return;
   }
   System.out.println("Ngay thang nam nhap vao hop le - Day la Nam Nhuan");
```